openid-sharedsignals-framework-1_0.md

```diff
@@ -1462,9 +1452,9 @@ Errors are signaled with HTTP status codes as follows:
 
 Examples:
 
-1. If a Receiver makes a request with an invalid OAuth token, then the
+1. If a Receiver makes a request with an invalid token, then the
    Transmitter MUST respond with a 401 error status.
-2. If the Receiver presents a valid OAuth token, but the Transmitter policy
+2. If the Receiver presents a valid token, but the Transmitter policy
```

Comment:
Suggest changing this to: "If the Receiver presents a valid authorization, but the..."